yarn.lock

```diff
@@ -244,6 +244,7 @@ __metadata:
     "@types/totp-generator": ^0.0.3
     "@wordpress/i18n": ^4.2.4
     asana-phrase: ^0.0.8
+    got: 11.8.3
```

Comment:
`got` is now on v12.x.x but that drops CommonJS export, which causes problems when imported and used within `calypso-e2e`.